docs/dev-tools/grokdebugger/index.asciidoc

```diff
@@ -1,25 +1,114 @@
 [role="xpack"]
 [[xpack-grokdebugger]]
-== Debugging Grok Expressions
+== Debugging grok expressions
 
 Grok is a pattern matching syntax that you can use to parse arbitrary text and
-structure it. Grok is perfect for parsing syslog logs, apache and other
-webserver logs, mysql logs, and in general, any log format that is generally
-written for humans and not computer consumption.
+structure it. Grok is good for parsing syslog, apache, and other
+webserver logs, mysql logs, and in general, any log format that is
+written for human consumption.
 
 Grok patterns are supported in the ingest node
 {ref}/grok-processor.html[grok processor] and the Logstash
 {logstash-ref}/plugins-filters-grok.html[grok filter]. The Elastic Stack ships
-with over 120 reusable grok patterns. See
-https://github.com/elastic/elasticsearch/tree/master/modules/ingest-common/src/main/resources/patterns[Ingest node grok patterns] and https://github.com/logstash-plugins/logstash-patterns-core/tree/master/patterns[Logstash grok patterns]
+with more than 120 reusable grok patterns. See
+https://github.com/elastic/elasticsearch/tree/master/libs/grok/src/main/resources/patterns[Ingest node grok patterns] and https://github.com/logstash-plugins/logstash-patterns-core/tree/master/patterns[Logstash grok patterns]
 for the full list of patterns.
 
-You can build and debug grok patterns in the Grok Debugger tool in {kib}
+You can build and debug grok patterns in the *Grok Debugger* in {kib}
 before you use them in your data processing pipelines. Because
 ingest node and Logstash share the same grok implementation and pattern
-libraries, any grok pattern that you create in the Grok Debugger will work
+libraries, any grok pattern that you create in the *Grok Debugger* will work
 in ingest node and Logstash.
 
+[float]
+[[grokdebugger-getting-started]]
+=== Getting started with the Grok Debugger
+
+This example walks you through using the *Grok Debugger*. This tool
+is automatically enabled in {kib}. 
+
+NOTE: If you're using {security}, you must have the `manage_pipeline`
+permission to use the Grok Debugger.
+
+. In the side navigation, click *DevTools*, then open the *Grok Debugger*. 
+. In *Sample Data*, enter a message that is representative of the data that you
+want to parse. For example:
++
+[source,ruby]
+-------------------------------------------------------------------------------
+55.3.244.1 GET /index.html 15824 0.043
+-------------------------------------------------------------------------------
+
+. In *Grok Pattern*, enter the grok pattern that you want to apply to the data.
```

Comment:
Do we have any docs to link to on grok syntax?

Reply:
Perhaps https://www.elastic.co/guide/en/logstash/current/plugins-filters-grok.html#_grok_basics?